valhalla/tyr/actor.h

```diff
@@ -1,14 +1,17 @@
 #ifndef VALHALLA_TYR_ACTOR_H_
 #define VALHALLA_TYR_ACTOR_H_
 
-#include <valhalla/baldr/graphreader.h>
-#include <valhalla/proto/api.pb.h>
-
-#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/ptree_fwd.hpp>
```

Comment:
I hardly imagine how `actor.h` can be used without including `#include <boost/property_tree/ptree.hpp>` because of ctor argument `actor_t` takes, so this change only improves includes for files that don't use `actor_t` at all and where removing inclusion of the `actor.h` would be more beneficial.

Reply:
yeah true, I'll revert these changes

Reply:
reverted these